spec/features/reservations_spec.rb

```diff
@@ -545,4 +545,24 @@
       end
     end
   end
+  context 'redirects properly' do
+    before(:each) { empty_cart }
```

Comment:
We also want to be signed in for the first test since I think guests are forbidden from viewing the `/reservations/new` page.